src/openms/source/KERNEL/Feature.cpp

```diff
@@ -166,10 +166,12 @@ namespace OpenMS
   bool Feature::encloses(double rt, double mz) const
   {
     ConvexHull2D::PointType tmp(rt, mz);
-    for (vector<ConvexHull2D>::const_iterator   it = convex_hulls_.begin(); it != convex_hulls_.end(); ++it)
+    for (const ConvexHull2D it : convex_hulls_)
```

Comment:
Please rename variable name. Not an iterator anymore. E.g. just c or hull would be fone